releasenotes/notes/deprecation-passmanager-0.25-95eb9b45b517370a.yaml

```diff
@@ -0,0 +1,6 @@
+---
+upgrade:
+  - |
+    Removed deprecated function :func:`~qiskit.transpiler.preset_passmanagers.common.get_vf2_call_limit`
+    and its corresponding test. Instead, use
```

Comment:
One more small point for release note style is that we ought to only mention user-facing changes, so in this case we can omit the bit about the tests being updated. Not a reason to hold up this PR, we can easily adjust this when we create and review the release notes PR during release 🙂.

Reply:
Sounds good, I will keep this in mind the next time I write a release note, thanks! :-)